### DISCUSSION TITLE
Can't get Stride.AssetPack.Building from nuget.org

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#2085] [Q&A]
The article(https://doc.stride3d.net/latest/en/manual/nuget/consume-packages.html) shows the way to load assets from nuget rather than project Assets folder, but I can't get the package.

**Comment by Doprez** [comment]
which package are you trying to get, and does it show any errors?

**Reply by DJDQfff** [reply]
official docs showed a way to load default resource from nuget (package:Stride.AssetPack.Building),but I can't get it from nuget.org

**Reply by Doprez** [reply]
If it doesnt show any packages then it may be a nuget config issue or if it doesnt show `Stride.AssetPack.BuildingBlocks` then that may have just been a demo package for the docs, I dont see it either.

You can try to pull https://www.nuget.org/packages/Stride.Voxels its what you need for the voxel lighting if I remember correctly. If you need the building blocks you can create a temp project with the demo models and pull them into your project.
